cms/toolbar/utils.py

```diff
@@ -144,4 +192,7 @@ def get_object_structure_url(obj, language=None):
         language = get_language()
 
     with force_language(language):
-        return admin_reverse('cms_placeholder_render_object_structure', args=[content_type.pk, obj.pk])
+        url = admin_reverse('cms_placeholder_render_object_structure', args=[content_type.pk, obj.pk])
```

Comment:
I don't think the structure url should have this param, it's not visible to a user.